### DISCUSSION TITLE
How to ignore objects in "front" of required zone

### DISCUSSION THREAD

**Discussion by MikeExMachina** [blakeblackshear/frigate#2616] [Ask A Question]
I have a camera looking out at the front lawn. At the end of the lawn there is a sidewalk going horizontally across the frame, then a verge (strip of grass between sidewalk and street) and then the street. The access hole for our sprinkler system is in the verge (behind the sidewalk). We've had problems with people messing with it, and I'd like to monitor the region of the verge around the access hole. It gets tricky because from the cameras perspective, this region is right behind the sidewalk. 

Q: If I track "person" in this zone around the access hole, and add that zone to "required zones" for events for that camera, will people walking down the sidewalk (obscuring the cameras view of the access hole, but not actually stepping off the sidewalk into that region) trigger an event in the access hole zone? Phrased another way, what is the logic for determining if an object is present in a required zone? Will any overlap of the object bounding box with the required zone cause a trigger? Or is it more specific (e.g. the center point of the bottom edge of the bounding box has to enter the zone, if this is the case then it should just work the way I want it to), and is that behavior configurable?

Any guidance on how to properly configure frigate for this scenarios would be greatly appreciated.

**Comment by blakeblackshear** [comment]
Bottom center of the bounding box has to enter the zone. You will have to play with it to see if it will be accurate enough for a small area.
